antora/docs/modules/ROOT/templates/policy_configuration.hbs

```diff
@@ -180,20 +180,27 @@ file:
 }
 ----
 
-Currently there are not many "collections" defined, but in future there may be more.
-
-The predefined collections are:
+=== Available collections
 
 [cols="2,6"]
 |===
 |*Name*
 |*Description*
 
-| `slsa_1`
-| Includes policy rules required to meet SLSA Level 1
+| `minimal`
```

Comment:
This section could also be generated if `processCollectionLists` were to return an array and we'd have a `[minimal|slsa1|slsa2|slsa3].rego` files containing the descriptions on the package.

Reply:
Yeah, I would like to have new collections just appear in this list. I think it's doable, but will tackle it in a later PR I think.

Using a dummy rego package just for the collection descriptions annotations is an interesting new (to me at least) idea.

Reply:
I filed https://issues.redhat.com/browse/HACBS-1749 about this.